data/_ontologies/Genetic variation.csv

```diff
@@ -0,0 +1,31 @@
+Single Nucleotide Polymorphism in Coding Sequence,Single Nucleotide Polymorphism,ncit,NCIT_C18518,http://purl.obolibrary.org/obo/NCIT_C18518,"Single Nucleotide Polymorphisms in Coding Sequences consist of a variation at an appreciable frequency between individuals of a single interbreeding population of a single nucleotide, due to base substitution, at an equivalent location within a protein-coding region of a gene that may or may not cause an amino acid change in the protein product and a detectable phenotypic effect."
+Non-synonymous Coding SNP,Single Nucleotide Polymorphism in Coding Sequence,ncit,NCIT_C45397,http://purl.obolibrary.org/obo/NCIT_C45397,"A Non-synonymous Coding SNP consists of a variation at an appreciable frequency between individuals of a single interbreeding population of a single nucleotide, due to base substitution, at an equivalent location within a protein-coding region of a gene that causes an alteration of the translation of the affected codon into a different amino acid in the protein product. A missense non-synonymous SNP results in an amino acid change in the protein product due to a codon change, while a nonsense non-synonymous SNP results in a termination codon."
+Synonymous Coding SNP,Single Nucleotide Polymorphism in Coding Sequence,ncit,NCIT_C45396,http://purl.obolibrary.org/obo/NCIT_C45396,"Synonymous Coding SNP consists of a variation at an appreciable frequency between individuals of a single interbreeding population of a single nucleotide, due to base substitution, at an equivalent location within a protein-coding region of a gene that causes no amino acid change in the protein product and no detectable phenotypic effect; a silent mutation."
+Single-Stranded Conformational Polymorphism,Genetic Polymorphism,ncit,NCIT_C17762,http://purl.obolibrary.org/obo/NCIT_C17762,"A Single-Stranded Conformational Polymorphism consists of nucleotide sequence variation in the same region of DNA between individuals within a species, or between alleles in an individual, that cause partially denatured DNA fragments to take on different conformations as a result of their nucleotide sequences differences. These conformation differences can be detected through various gel-separation techniques."
+Restriction Fragment Length Polymorphism,Genetic Polymorphism,ncit,NCIT_C17093,http://purl.obolibrary.org/obo/NCIT_C17093,Variation occurring within a species in the length of DNA fragments generated by a specific endonuclease. Such variations are generated by mutations that create or abolish recognition sites for these enzymes.
+Variant Pathogenic/Likely Pathogenic,,ncit,NCIT_C168803,http://purl.obolibrary.org/obo/NCIT_C168803,An ad hoc category that describes genetic variants with 90% or greater certainty of contributing to the development of a disease.
```

Comment:
and this also seems out of place

Reply:
And the other classifcations in line 27-30

Reply:
Looks like there are conflicting ontology datasets. Which one should we use?

Reply:
Removed table. Created table for classification and variant type